### PR TITLE
Add overrides and SchedulingThroughputPrometheus to SLOMeasurement

### PR DESCRIPTION
**Pull request by marseel**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:
Adds support for overrides in SLOMeasurement and includes SchedulingThroughputPrometheus in SLOMeasurement.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: clusterloader2/pkg/measurement/common/slos/slo_measurement.go [hunk 1]
```diff
@@ -68,6 +72,23 @@ func (s sloMeasurement) Execute(config *measurement.Config) ([]measurement.Summa
 	return s.summaries, nil
 }
 
+func getMeasurementConfig(config *measurement.Config, measurement_name string) (*measurement.Config) {
+	measurementConfig := *config
+	measurementConfig.Params = util.CloneMap(config.Params)
+
+	overrides, err := util.GetMap(config.Params, "overrides")
+	if err == nil {
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/slos/slo_measurement.go:80]: You shouldn't silently ignoring the error.

Comment (wojtek-t) [reply clusterloader2/pkg/measurement/common/slos/slo_measurement.go:80]: In other words, this function should return (Config, error)

File: clusterloader2/pkg/measurement/common/slos/slo_measurement.go [hunk 2]
```diff
@@ -54,7 +55,10 @@ type sloMeasurement struct {
 func (s sloMeasurement) Execute(config *measurement.Config) ([]measurement.Summary, error) {
 	errList := errors.NewErrorList()
 	for _, m := range s.measurements {
-		summaries, err := m.Execute(config)
+
```
[the remark this answers is not in the document]
Comment (marseel) [reply clusterloader2/pkg/measurement/common/slos/slo_measurement.go:63]: Done.

File: clusterloader2/pkg/measurement/common/slos/slo_measurement.go (hunk 1 above)
Comment (marseel) [reply clusterloader2/pkg/measurement/common/slos/slo_measurement.go]: Done.

**Comment by wojtek-t** [comment]
/hold

I would like to see if enabling scheduling throughput in our test configs won't break anything.

**Comment by marseel** [comment]
@wojtek-t looks good.

**Comment by wojtek-t** [comment]
Scheduling throughput is 73:
https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/perf-tests/1529/pull-perf-tests-clusterloader2/1315962499295612928/artifacts/SchedulingThroughputPrometheus_load_2020-10-13T10:56:08Z.json

Is that expected?

**Comment by marseel** [comment]
Yes, the difference comes from the fact that prometheus metric is scraped every 5s (previous scheduling throughput interval in load test was 1s). For 100 nodes it takes around 7 seconds to create pods used for measuring scheduling throughput. Also, even with old SchedulingThroughput, this measurement was turned off (alerting) for 100 Nodes clusters.

**Comment by marseel** [comment]
I think we can expect results like this. SchedulingThroughput on 100 nodes is quite short, previous SchedulingThroughput was measuring with interval 1 s, this one based on prometheus metric is measuring with interval 5s. Also, it's worth to mention that for 100 nodes, alerting is turned off for SchedulingThroughput.

**Comment by wojtek-t** [comment]
OK - we should probably document this somewhere in the subsequent PR.
Another thing to do will be to get rid of the old scheduling throughput measurement.

/lgtm
/hold cancel

**Pull request merged**